pom.xml

```diff
@@ -27,6 +27,9 @@
             <include>**/*Test.java</include>
             <include>**/*Tests.java</include>
           </includes>
+          <systemPropertyVariables>
+            <worker.id>$${surefire.forkNumber}</worker.id>
```

Comment:
why is double `$` needed?

Reply:
See https://stackoverflow.com/a/57494619/1333061

Reply:
thx!